### PR TITLE
update ENV variable

### LINKED ISSUE #521: Developer Solutions: On your Ubuntu host, setup ENV for...
Choose one: BUG REPORT

**What happened**: ENV should be env

**What you expected to happen**: ENV to be updated to env

### PR DESCRIPTION
**Pull request by mahebbar**
**What this PR does / why we need it**: Resolves ENV variable to env variable.

**Which issue this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*: fixes #521

**Special notes for your reviewer**: None

### CODE REVIEW

File: documentation/source/install/dev_solutions.rst [hunk 1]
```diff
@@ -69,7 +69,7 @@ On your Ubuntu host, setup directories for storing minkube and kubectl configura
     mkdir $HOME/.kube || true
     touch $HOME/.kube/config
 
-On your Ubuntu host, setup ENV for minikube. Copy the following to ~/.profile.
+On your Ubuntu host, setup env for minikube. Copy the following to ~/.profile.
```
Comment (kmova) [inline documentation/source/install/dev_solutions.rst]: env -> environment

File: documentation/source/install/dev_solutions.rst [hunk 2]
```diff
@@ -141,7 +141,7 @@ Download the latest OpenEBS Operator files using the following commands.
    kubectl apply -f openebs-operator.yaml
 
 **Note** 
-By default, OpenEBS launches OpenEBS Volumes with two replicas. To set one replica, as is the case with single-node Kubernetes cluster, specify the ENV variable *DEFAULT_REPLICA_COUNT=1*. This is supported in OpenEBS version 0.4 onwards. 
+By default, OpenEBS launches OpenEBS Volumes with two replicas. To set one replica, as is the case with single-node Kubernetes cluster, specify the env variable *DEFAULT_REPLICA_COUNT=1*. This is supported in OpenEBS version 0.4 onwards.
```
Comment (kmova) [inline documentation/source/install/dev_solutions.rst]: env -> environment

File: documentation/source/install/dev_solutions.rst (hunk 1 above)
Comment (mahebbar) [reply documentation/source/install/dev_solutions.rst]: Missed this :(. Updated.

File: documentation/source/install/dev_solutions.rst (hunk 2 above)
Comment (mahebbar) [reply documentation/source/install/dev_solutions.rst]: Thanks! Done!

**Pull request merged**